project/src/main/webapp/components/user-home/docs-component.js

```diff
@@ -3,23 +3,27 @@ import {html, LitElement} from 'https://unpkg.com/@polymer/lit-element/lit-eleme
 export class DocsComponent extends LitElement {
   static get properties() {
     return {
-      documents: {type: Array},
+      documents: {type: Object},
     };
   }
 
   constructor() {
     super();
-    this.documents = [
-      {'title': 'A doc', 'timestamp': 'June 16, 5:28PM'},
-      {'title': 'Another doc', 'timestamp': 'June 15, 1:30PM'}
-    ]
+    this.documents = [];
   }
 
   // Remove shadow DOM so styles are inherited
   createRenderRoot() {
     return this;
   }
 
+  getServletData() {
+    /*fetch('/UserHome').then((response) => response.json()).then((documentList)
```

Comment:
Is this code still useful? Add a comment here if so (i.e. why it's commented out), otherwise delete it.

Reply:
I commented it out since the backend was throwing NullPointerExceptions; this is fixed in PR #27 and I will uncomment this code.